Setup/Shield-Changelog.md

```diff
@@ -5,40 +5,39 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-- Git Issues should be referenced by #
-- Main Features/Bug Fixes should have (*)
-- User Action Required should have (!)
-
-## [Rel-19.05:Build_xxx] - 2019-05-15
+## [Rel-19.05.52x] - 2019-05-22
 
 ### New Features
 
 - New Virtual Appliance (Shield and Registry) for Offline Deployments (OVA)
 - CDR: Support Named Policy
 - Page Load Time Improvements
+- Admin: UX Improvement
```

Comment:
please change to:
Admin: Categories merged to Policies Table
Admin: Applications & Policies - Defaults table merged into main table for improved UX.